add company contact tabs

defrance opened the pull request:
add contacs of another company (external provider, ...) to a third-party company 
displays the list of members of a society
appair only if MAIN_SUPPORT_CONTACT_TYPE_FOR_THIRDPARTIES is enabled

defrance committed:
Update company.lib.php

defrance committed:
add contact tabs on company

appair only if MAIN_SUPPORT_CONTACT_TYPE_FOR_THIRDPARTIES is enabled
display on tabs adherent list of company

defrance committed:
Update societecontact.php

defrance committed:
Update societecontact.php

defrance committed:
Update societecontact.php

eldy commented:
I my understanding is correct, does the following option name would not be better :+1: 
MAIN_SUPPORT_SHARED_CONTACT_BETWEEN_THIRDPARTIES
instead of
MAIN_SUPPORT_CONTACT_TYPE_FOR_THIRDPARTIES

or did  i not understood goal of PR ?

defrance commented:
Your understand is correct
I just prefered reuse an older active option name.
Matbe better to change it too...

eldy commented:
Good eyes.
This constant was never used and is present at one place only. So i prefer to have it renamed into MAIN_SUPPORT_SHARED_CONTACT_BETWEEN_THIRDPARTIES.
Can you complete your PR with this ?

defrance committed:
change option name

replace
MAIN_SUPPORT_CONTACT_TYPE_FOR_THIRDPARTIES
by
MAIN_SUPPORT_SHARED_CONTACT_BETWEEN_THIRDPARTIES
